### DISCUSSION TITLE
Rowy hangs on project loading screen

### DISCUSSION THREAD

**Discussion by ivantodorov78** [AntlerVC/firetable#1489] [Support / Q&A]
Hi there, when I click on my project link and rowy hangs on 80% loading screeen w/o any explanation. Please LMK any ideas on what to do, and thank you. 

<img width="204" alt="• Rowy 2023-11-21 08-32-17" src="https://github.com/rowyio/rowy/assets/18202192/ff7081a0-7ba3-4e6f-87f2-4c49dce01cea">

**Comment by Chadha93** [comment]
Hi @ivantodorov78, could you please add a jam.dev video for this?

**Reply by Chadha93** [reply]
Closing this, feel free to reopen if issue still exists.
